examples/multisig/lib.rs

```diff
@@ -135,6 +138,8 @@ mod multisig {
         pub transferred_value: Balance,
         /// Gas limit for the execution of the call.
         pub gas_limit: u64,
+        /// The call flag allowing reentry
+        pub allow_reentry: bool,
```

Comment:
Better to pass the `CallFlag` itself to Allow more control:)
Also, we can add a note that allowing reentrancy may cause vulnerabilities

Reply:
Good idea.

Reply:
Well, this would require implementing `parity_scale_codec::WrapperTypeEncode` for `ink_env::CallFlags` which is out of the scope of this contract. While this PR is just a bugfix of the contract.